### PR TITLE
SG-7: Photo upload function with DB

### PR DESCRIPTION
**Pull request by Serikbolkz**
Photo upload functionality is added. Title, description and image are stored in database. Images are stored in image folder 

DB
Image folder
Upload Form

### CODE REVIEW

File: base/migrations/0002_alter_image_image.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.3 on 2023-07-07 12:49
+
+from django.db import migrations, models
```
Comment (stalgatuly) [inline base/migrations/0002_alter_image_image.py:3]: alter_image_image?

File: image/2022-10-26_5.png
Comment (stalgatuly) [inline image/2022-10-26_5.png:1]: Add database folder to gitignore

File: base/migrations/0002_alter_image_image.py (hunk 1 above)
Comment (stalgatuly) [reply base/migrations/0002_alter_image_image.py:3]: why 0001_initial? why 0002?

**Review by stalgatuly — CHANGES_REQUESTED** [review]

**Comment by stalgatuly** [comment]
Resolve conflicts, next time make a branch not from master, but from a branch that is under review

You can resolve them here or resolve locally and make a push with --force flag.

**Comment by stalgatuly** [comment]
Overall, good job!

File: base/migrations/0002_alter_image_image.py (hunk 1 above)
Comment (Serikbolkz) [reply base/migrations/0002_alter_image_image.py:3]: > alter_image_image?

@sagynbek001 It was created and named automatically when I have changed database.

Comment (Serikbolkz) [reply base/migrations/0002_alter_image_image.py:3]: > why 0001_initial? why 0002?

I did not named them. When I wrote "python3 manage.py makemigrations" in ubuntu to update database, they were created by program

Comment (stalgatuly) [reply base/migrations/0002_alter_image_image.py:3]: Okay, but they sound weird. Maybe there is a flag when creating to put a chosen name. Take look next time.

**Review by stalgatuly — APPROVED** [review]

**Pull request merged**